aksel.nav.no/Dockerfile

```
@@ -5,6 +5,7 @@ ENV NODE_ENV production
 
 WORKDIR /app
 COPY aksel.nav.no/website/.next/standalone ./
+COPY next-logger.config.js ./
```

Tror full path må være med
```suggestion
COPY aksel.nav.no/website/next-logger.config.js ./
```

---

Ah, good catch!